mvc-api/mvc-service/app/managers/EventObjectProducer.java

```diff
@@ -0,0 +1,37 @@
+package managers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.simple.JSONObject;
+import org.sunbird.search.util.SearchConstants;
+
+import java.util.LinkedHashMap;
+import java.util.UUID;
+
+public class EventObjectProducer {
+    EventProducer eventProducer = new EventProducer();
+    public void addToEventObj(JSONObject content, String contentId)  {
+        // Add label
+        content.put("label","MVC");
+        ObjectMapper event = new ObjectMapper();
```

Comment:
Don't create this inside method. There is JsonUtils.java class. Please use that for any `Object` serialization and deserialization.